flake.nix

```diff
@@ -127,10 +139,11 @@
           rustfmt = let
             wrapper = final.symlinkJoin {
               name = "rustfmt-wrapped";
-              paths = [final.rustfmt];
+              paths = [rustfmt-nightly];
               nativeBuildInputs = [final.makeWrapper];
               postBuild = let
-                PATH = final.lib.makeBinPath [final.cargo final.rustfmt];
+                # Use nightly rustfmt
+                PATH = final.lib.makeBinPath [final.fenix.stable.cargo rustfmt-nightly];
               in ''
                 wrapProgram $out/bin/cargo-fmt --prefix PATH : ${PATH};
               '';
```

Comment:
@garbas d you remeber why we had to add our own wrapper around rustfmt?
I think it was due to not being able to set rustfmt using the `tools` config but I'm not sure.

Reply:
indeed since https://github.com/cachix/git-hooks.nix/pull/397 that should be possible without a custom wrapper